docs/modules/ROOT/partials/getting-started/con-registry-security.adoc

```diff
@@ -0,0 +1,62 @@
+// Metadata created by nebel
+
+[id="registry-security"]
+
+= Configuring {registry} with Authentication
+
+This section explains how to configure authentication and authorization for {registry}. In order to configure the authentication the following environment variables should be used:
+
```

Comment:
Suggest "This section explains how to configure authentication and authorization for {registry} based on OpenID Connect (OIDC)."